composer.json

```diff
@@ -10,15 +10,22 @@
 			"homepage": "http://www.matej21.cz"
 		}
 	],
+	"repositories": [
+		{
+			"type": "git",
+			"url": "https://github.com/paveljurasek/events"
+		}
+	],
 	"require": {
-		"nette/utils": "~2.4",
-		"nette/di": "~2.4",
-		"kdyby/doctrine": "@dev",
-		"kdyby/events": "@dev"
+		"php": "^7.1",
+		"nette/utils": "^3.0",
+		"nette/di": "^3.0-RC3",
```

Comment:
we should definitely wait until nette conference for stable tags

Reply:
Sure, just let's let it hang here until stables are released.